app_server/.eslintrc

```diff
@@ -0,0 +1,72 @@
+{
+  // http://eslint.org/docs/rules/
+
+  "env": {
+    "browser": true,      // browser global variables.
+    "node": true,        // Node.js global variables and Node.js-specific rules.
+    "worker": false,      // web workers global variables.
+    "amd": false,         // defines require() and define() as global variables as per the amd spec.
+    "mocha": false,       // adds all of the Mocha testing global variables.
+    "jasmine": false,     // adds all of the Jasmine testing global variables for version 1.3 and 2.0.
+    "phantomjs": false,   // phantomjs global variables.
+    "jquery": false,      // jquery global variables.
+    "prototypejs": false, // prototypejs global variables.
+    "shelljs": false,     // shelljs global variables.
+    "meteor": false,      // meteor global variables.
+    "mongo": false,       // mongo global variables.
+    "applescript": false, // applescript global variables.
+    "es6": false,         // enable all ECMAScript 6 features except for modules.
```

Comment:
Yup, I got it those from a rc for google style guide so just leave it there in case we need to switch something on/off. Anyway, I will clean those part up then.